tests/transform_dialect/cpu/fmaxf_16bit_expantion.mlir

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
You need to trigger the iree-opt run and FileCheck, see other `.mlir` file for examples.